scripts/run_lighthouse_tests.py

```diff
@@ -57,51 +51,27 @@
 """)
 
 _PARSER.add_argument(
-    '--mode',
-    help='Sets the mode for the lighthouse tests',
-    required=True,
-    choices=['accessibility', 'performance'],)
-
-
-def cleanup():
-    """Deactivates webpages and deletes html lighthouse reports."""
-    pattern = '"ENABLE_ACCOUNT_DELETION": .*'
-    replace = '"ENABLE_ACCOUNT_DELETION": false,'
-    common.inplace_replace_file(common.CONSTANTS_FILE_PATH, pattern, replace)
-
-    build.set_constants_to_default()
-
-    google_app_engine_path = '%s/' % common.GOOGLE_APP_ENGINE_SDK_HOME
-    processes_to_kill = [
-        '.*%s.*' % re.escape(google_app_engine_path),
-    ]
-    for p in SUBPROCESSES:
-        p.kill()
-
-    for p in processes_to_kill:
-        common.kill_processes_based_on_regex(p)
-
-    common.stop_redis_server()
+    '--mode', help='Sets the mode for the lighthouse tests',
+    required=True, choices=['accessibility', 'performance'],)
 
 
 def run_lighthouse_puppeteer_script():
     """Runs puppeteer script to collect dynamic urls."""
-    puppeteer_path = os.path.join(
-        'core', 'tests', 'puppeteer', 'lighthouse_setup.js')
+    puppeteer_path = (
+        os.path.join('core', 'tests', 'puppeteer', 'lighthouse_setup.js'))
```

Comment:
Is there a stylistic reason to prefer this? To me it seems like it just adds unnecessary parentheses, but I've seen you do it twice so I assume it's on purpose

Reply:
Yep, I do it because I prefer entire units of code to appear on one line (function calls, strings, expressions, etc.). I think this is officially known as "The Rectangle Rule": https://github.com/google/google-java-format/wiki/The-Rectangle-Rule.

There doesn't seem to be any enforcement on how to break lines in our linters, so this is how I've prefered handling them.

Reply:
I don't mind either approach, but in general I would probably suggest leaving stuff as-is by default since this PR is large enough already (and stylistic changes that aren't really needed make it a bit harder to review).